src/qibolab/instruments/qm.py

```diff
@@ -254,7 +254,7 @@ def register_drive_element(self, qubit, intermediate_frequency=0):
             # register drive controllers
             self.register_analog_output_controllers(qubit.drive.ports)
             # register element
-            lo_frequency = qubit.drive.lo_frequency
+            lo_frequency = int(qubit.drive.local_oscillator.frequency)
```

Comment:
Do you really need the `int` conversion? Why?

Reply:
I believe so, because I noticed that the local oscillators (at least the Rohde-Schwarz ones) support float frequencies, while QM instruments do not and it will complain if I pass a float in their config.